src/crimson/os/seastore/seastore_types.h

```diff
@@ -329,23 +329,22 @@ std::ostream &operator<<(std::ostream &out, const paddr_list_t &rhs);
  */
 enum class extent_types_t : uint8_t {
   ROOT = 0,
-  LADDR_INTERNAL = 1,
```

Comment:
These are on-disk values, the numerical values need to be specified.

Reply:
I guess you want to use them as indices into arrays.  I'm fine with renumbering them to be dense.

Reply:
Yeah, added numerical values back.